analyzer/codechecker_analyzer/analyzers/clangsa/statistics_collector.py

```diff
@@ -7,12 +7,10 @@
 """
 Calculates call statistics from analysis output
 """
-from __future__ import division
-from __future__ import print_function
-from __future__ import absolute_import
+
 
 try:
```

Comment:
Are we planning to be python3 only? In that case one of the imports are redundant here.

Reply:
Maybe we should grep for `ImportError` as this pattern might be there at other places as well.

Reply:
Yes we are planning to be python3 only and CodeChecker will require at least python 3.6.
I've cleaned up the imports.